openchs-server-api/src/main/java/org/openchs/service/IndividualService.java

```diff
@@ -84,9 +96,21 @@ public IndividualContract getSubjectInfo(String individualUuid) {
         }).collect(Collectors.toList());
     }
 
+    public Set<EncounterContract> constructEncounters(Set<Encounter> encounters) {
+        return encounters.stream().map(programEncounter -> {
+            EncounterContract encountersContract = new EncounterContract();
+            encountersContract.setUuid(programEncounter.getUuid());
+            encountersContract.setName(programEncounter.getName());
+            encountersContract.setOperationalEncounterTypeName(programEncounter.getEncounterType().getOperationalEncounterTypeName());
+            encountersContract.setEncounterDateTime(programEncounter.getEncounterDateTime());
+            encountersContract.setEarliestVisitDateTime(programEncounter.getEarliestVisitDateTime());
+            encountersContract.setMaxVisitDateTime(programEncounter.getMaxVisitDateTime());
```

Comment:
map the voided field as well. Raise a separate PR to do it also in all the new apis that you have written.